platforms/r3-corda/configuration/roles/create/k8_component/templates/create_doorman.tpl

```diff
@@ -15,6 +15,9 @@ spec:
     nodeName: {{ services.doorman.name }}
     metadata:
       namespace: {{component_ns }}
+      component_name: {{ services.doorman.name }}
```

Comment:
Add corresponsing keys in Values.yaml as well for all the charts. With proper examples.